nntrainer/layers/layer.cpp

```diff
@@ -363,7 +363,7 @@ void LayerV1::print(std::ostream &out, unsigned int flags) {
   }
 };
 
-std::shared_ptr<LayerV1> getLayerDevel(std::shared_ptr<ml::train::Layer> l) {
+std::shared_ptr<LayerV1> getLayerV1Devel(std::shared_ptr<ml::train::Layer> l) {
```

Comment:
I quite dont understand why it needs getLayerDevel?? is there any reason?? If it is getLayerDevel.. then do we have getLayer as well?? and does it return different??

Reply:
This is for internal use only. This is a wrapper function for LayerNode::getObject() and this will be removed once getObject() is removed.

Reply:
I think this is a temporary code when we were building `getLayerDevel`
@kparichay any opinions?